examples/vpc-scenario-2/main.tf

```diff
@@ -188,25 +188,6 @@ docker run                   \
 END_INIT
 }
 
-#resource "aws_instance" "bastion" {
```

Comment:
@ketzacoatl This section (L191 - L209) was commented out so I felt okay deleting it. I saw that you left it out as well in your commits. I figured it was residual work from whoever originally wrote the example.